bigbluebutton/greenlight
# Allow user to view their rooms

closes #3162 






## Description

- When a user goes to /rooms page, they are able to view the rooms that belong to them
- Each box contains room name, last session (to be integrated still), copy link icon (to be integrated still), and start room button (to be integrated still)
- Each card is clickable which takes you to the specifc room page rooms/:friendly_id
- RSpec test for rooms index get request


- NOTE: 
  - This is not the final look, but I tried to make the overall idea look similar to the mock ups to make it easier in the future
  - current_user needs to be implemented for the rooms/index to return dynamically and for the rspec test to work dynamically as well

## Testing Steps

Steps will change slightly when current_user is implemented
1. Pull branch and code
2. Open rails console
3. Create user with id:1
4. Create multiple rooms with user_id equal the to id of the user you created in step 3 (id:1)
5. Go to /rooms path, rooms should be visible here


## Screenshots (if appropriate):

![image](https://user-images.githubusercontent.com/38328371/160171593-6afef7ef-2295-4dcb-bfd7-5ffcf352b943.png)
